.github/workflows/integration-template-upload.yaml

```diff
@@ -1,14 +1,10 @@
 name: Upload integration templates to S3
 
 on:
+    workflow_dispatch:
     push:
         branches:
             - master
-    pull_request:
-        branches:
-            - master
-        types:
-            - closed
```

Comment:
I think the pull request trigger is redundant. We want to upload when a commit is pushed to master, that's it.